addons/web_editor/static/src/js/wysiwyg_snippets/wysiwyg_snippets.js

```diff
@@ -150,6 +150,7 @@ Wysiwyg.include({
         if (this.snippets) {
             this.snippets.updateCurrentSnippetEditorOverlay();
         }
+        this._summernote.layoutInfo.editable.mouseup();
```

Comment:
Well I will trust you on the code as it is the editor... but I find strange to apparently trigger mouseup by hand.

Reply:
Ok I ended up doing that and it actually fixes the rest as well so good call.
The line is now simply `this._summernote.invoke('toolbar.update', true);`